documentation/developers/Cookies.md

```diff
@@ -0,0 +1,35 @@
+﻿# Cookie Documentation
+
```

Comment:
Please add brief intro re: purpose of this document

Reply:
done.